app/views/vm_common/console_vmrc.html.haml

```diff
@@ -4,6 +4,7 @@
     - if %w(5.1 5.5).include?(api_version) && is_browser_ie? && browser_info(:version).to_i > 9
       %meta{:content => "IE=8", "http-equiv" => "X-UA-Compatible"}
     %meta{:content => "text/html; charset=ISO-8859-1", "http-equiv" => "Content-Type"}
+    = tag :link, :rel => "shortcut icon", :href => image_path("favicon.ico")
```

Comment:
Is it possible to use [favicon_link_tag](http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/favicon_link_tag) helper there?